src/main.rs

```diff
@@ -29,16 +29,17 @@ async fn main() {
         }
     }
     let mut output = String::from("Happy Birthday to ");
-    if users.len() == 1 {
-        output.push_str(&format!("{}!", users[0]));
-    } else if users.len() == 2 {
-        output.push_str(&format!("{} and {}!", users[0], users[1]));
-    } else {
-        for i in 0..users.len()-1 {
-            let s = &users[i];
-            output.push_str(&format!("{}, ", s));
+    match users.len() {
+        0 => output.push_str("no one. :sad-bidoof:");
+        1 => output.push_str(&format!("{}!", users[0]));
+        2 => output.push_str(&format!("{} and {}!", users[0], users[1]));
+        user_count => {
+            for i in [..user_count-1] {
+                let s = &users[i];
+                output.push_str(&format!("{:?}, ", s));
```

Comment:
Why did you turn this into a debug string?

Reply:
Uh... what is a debug string? 
Except for replacing `users.len()` with `user_count`, the code should be identical to the current >2 user case.

Reply:
After looking at the current master and my branch, it seems that the `{:?}` was left in master, until you removed it with the most recent commit to your master. When I resolved the merge conflict, I re-introduced it.